tsconfig.json

```diff
@@ -31,11 +31,21 @@
       "react"
     ]
   },
-  "include": [],
+  "include": [
+    "e2e/__helpers__",
+    "e2e/__serializers__",
+    "e2e/__tests__",
+    "scripts/",
+    "src",
+    "presets",
+    "./*.js"
+  ],
   "exclude": [
-    "e2e/__cases__",
-    "e2e/__templates__",
-    "e2e/__workdir_synlink__",
-    "node_modules"
+    // "e2e/__cases__",
```

Comment:
`exclude` isn't possible to leave as empty array ?

Reply:
I forgot that one, was during the test. Removed now in latest commit!